jobs/settings.py

```diff
@@ -10,6 +10,7 @@
 
 env = environ.Env()
 
+BASE_URL = env("BASE_URL", default="http://localhost:8000")
```

Comment:
localhost? Or app?

Reply:
Use localhost will support both developments (local and Docker)

// Base composition exposes `tcp#8000`, so will also work

Anyway, if needed can be overrided using `.env`